src/components/close-button/CloseButton.js.flow

```diff
@@ -9,15 +9,17 @@ import { bdlGray65 } from '../../styles/variables';
 import './CloseButton.scss';
 
 type Props = {
+    /** ariaLabel defines a string value that labels the current element */
+    ariaLabel?: string;
     /** Custom class for the close button */
     className?: string,
     /** onClick handler for the close button */
     onClick?: Function,
 }
 
-const CloseButton = ({ className, onClick }: Props) => {
+const CloseButton = ({ className, onClick, ariaLabel = 'close' }: Props) => {
```

Comment:
nit: these props are sorted alphabetically

Reply:
It's been removed. Thanks!